python/pyspark/tests/test_taskcontext.py

```diff
@@ -282,6 +281,11 @@ def context(iterator):
         self.assertTrue(bps == (-1, -1))
         for pid in pids:
             self.assertTrue(pid in worker_pids)
+        return True
+
+    def test_task_context_correct_with_python_worker_reuse(self):
+        eventually(
+            self.check_task_context_correct_with_python_worker_reuse, catch_assertions=True)
```

Comment:
nit: we may add few comment like the one in the description, e.g. "Retrying the check as reusing Python workers actually sort of makes the best effort reuse the Python workers."

Reply:
Sure sure 👌

Reply:
thank you :)